bin/mixer_codegen.sh

```diff
@@ -198,7 +198,7 @@ if [ "$opttemplate" = true ]; then
   fi
 
   templateYaml=${template/.proto/.yaml}
-  go run "$GOPATH/src/istio.io/istio/mixer/tools/mixgen/main.go" template -d "$templateSDS" -o "$templateYaml" -n "$(basename "$(dirname "${template}")")"
+  go run $GOPATH/src/istio.io/istio/mixer/tools/mixgen/main.go template -d $templateSDS -o $templateYaml -n $(basename "$(dirname "${template}")")
```

Comment:
Ditto.

Reply:
Same as below

Reply:
I don't believe this one should have any issues when quoted.

Reply:
Fixed